compositional/1D_COMP_NO_WELLS.DATA

```diff
@@ -0,0 +1,214 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2024, SINTEF Digital, TNO
+
+-- This is a 1D, pressure-driven CO₂ flooding example involving three components and two phases.
+-- The components are CO₂, methane, and decane.
+-- The first and last cells are assigned very large pore volumes to emulate a source and sink,
```

Comment:
Maybe instead of saying "The first and last cells..." we could say "Cells (1,1,1) and (30,1,1)..." in order to be very explicit about the coordinates?